runtime/src/chpl-file-utils.c

```diff
@@ -186,11 +186,13 @@ qioerr chpl_fs_is_mount(int* ret, const char* name) {
     // itself.
 
     char* curTok = strtok(safeNameCopy, "/");
-    // should never be null.  The only string which would return null is "/",
-    // but that directory is not a link, so won't be here in the first place.
     char* nextTok = strtok(NULL, "/");
     // We need the next token to determine if the path is longer than a single
     // link name.
+    assert(curTok != NULL);
```

Comment:
Can't.  Compilation complains about "mixed declarations and code".  Stupid pre-C99